src/Inputs/scene.inputManager.ts

```diff
@@ -861,35 +858,32 @@ export class InputManager {
                 evt.x = pointerX;
                 evt.y = pointerY;
 
-                if (attachDown && inputIndex >= PointerInput.LeftClick && inputIndex <= PointerInput.RightClick && currentState === 1) {   // Pointer Down
+                if (attachDown && inputIndex >= PointerInput.LeftClick && inputIndex <= PointerInput.RightClick && currentState === 1) {
+                    // Pointer Down
                     evt.type = "pointerdown";
-                    evt.button = (inputIndex - 2);
+                    evt.button = inputIndex - 2;
 
                     let currentEvent = eventData || evt;
 
-                    this._onPointerDown((currentEvent as IPointerEvent));
+                    this._onPointerDown(currentEvent as IPointerEvent);
                 }
 
-                if (attachUp && inputIndex >= PointerInput.LeftClick && inputIndex <= PointerInput.RightClick && currentState === 0) {   // Pointer Up
+                if (attachUp && inputIndex >= PointerInput.LeftClick && inputIndex <= PointerInput.RightClick && currentState === 0) {
+                    // Pointer Up
                     evt.type = "pointerup";
-                    evt.button = (inputIndex - 2);
+                    evt.button = inputIndex - 2;
                     let currentEvent = eventData || evt;
 
-                    this._onPointerUp((currentEvent as IPointerEvent));
+                    this._onPointerUp(currentEvent as IPointerEvent);
                 }
 
                 if (attachMove) {
-                    if (inputIndex === PointerInput.Horizontal
-                        || inputIndex === PointerInput.Vertical
-                        || inputIndex === PointerInput.DeltaHorizontal
-                        || inputIndex === PointerInput.DeltaVertical
-                        || inputIndex === PointerInput.FakeMove) {
+                    if (inputIndex === PointerInput.Horizontal || inputIndex === PointerInput.Vertical || inputIndex === PointerInput.DeltaHorizontal || inputIndex === PointerInput.DeltaVertical || inputIndex === PointerInput.FakeMove) {

```

Comment:
nit: same here, always a fan of multi-lining longer conditional checks.

Reply:
That's the auto-formatter causing havoc. Let me see if I can reconfigure it